{"name": "test_ceshi_002", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 < 1", "trace": "self = <Test.TestCase.test_ceshi.TestCeshi object at 0x7fad8e6d35b0>\n\n    @allure.story(\"这是测试1<1失败的情况\")\n    def test_ceshi_002(self):\n        print(\"这是测试用例1\")\n>       assert 1 < 1\nE       assert 1 < 1\n\nTest/TestCase/test_ceshi.py:19: AssertionError"}, "start": 1650044181929, "stop": 1650044181930, "uuid": "db10a26f-4929-4657-a010-ce1bd5a5ff22", "historyId": "ab0990ae58dd4c039f227e0eb85aedc7", "testCaseId": "72f1c64968fc2a4c71c17db7a4a099d5", "fullName": "Test.TestCase.test_ceshi.TestCeshi#test_ceshi_002", "labels": [{"name": "feature", "value": "ceshi"}, {"name": "story", "value": "这是测试1<1失败的情况"}, {"name": "parentSuite", "value": "Test.TestCase"}, {"name": "suite", "value": "test_ceshi"}, {"name": "subSuite", "value": "TestCeshi"}, {"name": "host", "value": "fv-az78-496"}, {"name": "thread", "value": "2511-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Test.TestCase.test_ceshi"}]}